lib/loaders/total.js

```diff
@@ -20,7 +20,7 @@ export const total = (path, options = {}) => {
 export const totalLoader = httpLoader(total);
 
 const load = (path, options = {}) =>
-  totalLoader.load(path, options)
+  totalLoader.load({ path, options })
```

Comment:
I'm confused as to why I was passing this options in the first place: I believe this only accepts a string key?